CHANGELOG.md

```diff
@@ -0,0 +1,97 @@
+# Changelog
+
+## [Unreleased]
+
+## [v0.1.5] - 2021-01-05
+
+### Added
+
+- Allow support for semicolumn and tabulation in CSV files
+- Collect current distance from OR-Tools
+- Allow initial routes when clustering methods are required
```

Comment:
we don't have this yet :thinking:

Reply:
This is allowed, it didn't ensure that missions of a single route are within the same cluster :/

Reply:
We still don't ensure that missions of a single route are within the same cluster in general. For example clustering based methods split_solve and dicho fail when there is initial_routes (optim-insert or other).

I suspect that you mean "partitions" (`[:preprocessing][:partitions]`), no?

Reply:
Yes, I should remove the ambiguity. And add a comment to mention, that even if it is allowed, we currently cannot ensure that members of a single route are within the same cluster.